Components/Grid.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+
+import '../css/Components/Grid';
+
+import GridColumn from './GridColumn';
+
+class Grid extends Component {
+  static propTypes = {
+    children: PropTypes.any,
+    offset: PropTypes.string,
+  };
+
+  render() {
+    const className = classNames("Grid", {
+      [`Grid--offset${this.props.offset}`]: this.props.offset,
+    });
+
+    return <div className={className}>{this.props.children}</div>
+  }
+}
+
+Grid.Column = GridColumn;
```

Comment:
I explain [here](https://github.com/subvisual/2017.mirrorconf.com/pull/8#discussion_r102087968) why I created a `Grid.Column`. If you were wondering why the name of `GridColumn`, it was because  the current project structure wasn't namespacing components inside specific folders (e.g. `Components/Grid/index.js` or `Components/Grid/Column.js`). Since that component is specific to the `Grid`, but open to other components, I decided to name it `GridColumn`.